cmd/nerdctl/image_history.go

```diff
@@ -58,11 +59,16 @@ func addHistoryFlags(cmd *cobra.Command) {
 		return []string{"json"}, cobra.ShellCompDirectiveNoFileComp
 	})
 	cmd.Flags().BoolP("quiet", "q", false, "Only show numeric IDs")
+	cmd.Flags().BoolP("human", "H", true, "Print sizes and dates in human readable format (default true)")
```

Comment:
Please sync https://github.com/containerd/nerdctl/blob/main/docs/command-reference.md

Reply:
> Please sync https://github.com/containerd/nerdctl/blob/main/docs/command-reference.md

Yep, always forget about these. Sorry about that. Next push will have it.